backtracepython/report.py

```diff
@@ -155,9 +154,18 @@ def log(self, line):
             }
         )
 
+    def add_attachment(self, attachment_path):
+        self.attachments.append(attachment_path)
+
+    def get_attachments(self):
+        return self.attachments
+
+    def get_data(self):
+        return self.report
+
     def send(self):
         if len(self.log_lines) != 0 and "Log" not in self.report["annotations"]:
             self.report["annotations"]["Log"] = self.log_lines
-        from backtracepython.client import send_worker_report
+        from backtracepython.client import send
```

Comment:
can we use global imports?

Reply:
unfortunately, this would require breaking the existing API. Our current interface allows the record to be sent directly via the send method. Since client has to have a reference to report (creating a report on unhandled exception or sending a message), and report has a reference to the client due to the send method, we have a circular reference